test/lower_triangular_matrix.jl

```diff
@@ -597,7 +597,7 @@ end
                 L2 = deepcopy(L1) 
 
                 L2 ./= 5
-                @test L1/5 == L2
+                @test L1/5 ≈ L2
```

Comment:
Given this, I'm wondering whether we should actually do `L/s = L*inv(s)` because the latter introduces two rounding errors. The latter is what an algorithm should implement for speed, but maybe we should move the method for exactly this reason here.